shared/templates/static/ansible/ensure_gpgcheck_never_disabled.yml

```diff
@@ -0,0 +1,19 @@
+# platform = multi_platform_rhel
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+#
+- name: Find All Yum Repositories
+  find:
+    paths: "/etc/yum.repos.d/"
+    patterns: "*.repo"
+  register: yum_find
+
+- name: Ensure gpgcheck Enabled For All Yum Package Repositories
+  with_items: "{{ yum_find.files }}"
+  replace:
+    dest: "{{ item.path }}"
+    regexp: 'gpgcheck = 0'
```

Comment:
shouldn't it use lineinfile instead? in case gpgcheck is not in the file? What if it's commented? What if it uses different whitespace around the key value pair?